levenshtein/php/code.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+// Main function to mimic C program's behavior
+function main(array $args): void
+{
+    $minDistance = -1;
+    $times = 0;
+
+    foreach ($args as $i => $str1) {
+        foreach ($args as $j => $str2) {
+            if ($i !== $j) {
+                $distance = levenshtein($str1, $str2);
```

Comment:
We need to actually implement `levenshtein` in this code rather than relying on the standard library function. It's possible they are not using the same exact algorithm as described in the [levenshtein readme](https://github.com/bddicken/languages/tree/main/levenshtein#readme). Can add an implementation here?

Reply:
@bddicken Ad.1 according to php "c"-source they behave the same (php have some improvements on memory management, but not a speed). For reference - https://github.com/php/php-src/blob/master/ext/standard/levenshtein.c Ad.2 Sure, I will refactor to pure implementation if still necessary ;

Reply:
Interesting. I'd like to actually have the implementations in the repo so that we are actually seeing what the php interpreter / jit is capable of, rather than a pre-compile / optimized implementation in C